### PR TITLE
elsevier: restrict records harvested from elsevier

### PR DESCRIPTION
**Pull request by MJedr**
Ref: inspirehep/inspirehep#1446

### CODE REVIEW

**Review by michamos — APPROVED** [review]
minor comment, but looks good :+1:

File: docker-compose.test.py2.yml [hunk 1]
```diff
@@ -199,7 +199,7 @@ services:
       test: "redis-cli -h 127.0.0.1 ping| grep PONG"
 
   localstack:
-    image: localstack/localstack:latest
+    image: localstack/localstack:0.11.6
```
Comment (drjova) [inline docker-compose.test.py2.yml:202]: 👍

File: docker-compose.test.py2.yml [hunk 2]
```diff
@@ -213,7 +213,7 @@ services:
       timeout: 5s
       interval: 5s
       retries: 5
-      test: "curl -k localhost:4566|grep running"
+      test: "curl -k localstack:4566|grep running"
```
Comment (drjova) [inline docker-compose.test.py2.yml:216]: do you we need to change to `localstack`?

Comment (MJedr) [reply docker-compose.test.py2.yml:216]: We set the `HOSTNAME_EXTERNAL=localstack` and `HOSTNAME=localstack` so I think that we should use localstack here (locally when I test with `docker-compose` the log is `Unable to find forwarding rule for host "localhost:4566"`)

**Pull request merged**